scala/grpc/src/main/java/com/intel/analytics/bigdl/grpc/GrpcClientBase.java

```diff
@@ -73,11 +79,9 @@ public ManagedChannel getChannel() {
     public void build() throws IOException {
         parseConfig();
         if (channel == null) {
-            channel = ManagedChannelBuilder.forTarget(target)
-                    .maxInboundMessageSize(Integer.MAX_VALUE)
-                    // Channels are secure by default (via SSL/TLS).
-                    .usePlaintext()
-                    .build();
+            channel = NettyChannelBuilder.forTarget(target)
```

Comment:
Need to add normal TLS without privateKeyFilePath

Reply:
Current: if `privateKeyFilePath` is `null` then use normal tls.